aptos-move/aptos-vm/src/move_vm_ext/vm.rs

```diff
@@ -93,6 +94,48 @@ impl MoveVmExt {
         })
     }
 
+    pub fn new_abstract_usage(
```

Comment:
Let's try to merge this with `Self::new` so you don't end up duplicating the configs here.